emop-maven-plugin/src/main/java/edu/cornell/RpsRppVmsMojo.java

```diff
@@ -0,0 +1,16 @@
+package edu.cornell;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Execute;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+@Mojo(name = "rps-rpp-vms", requiresDirectInvocation = true, requiresDependencyResolution = ResolutionScope.TEST)
+@Execute(phase = LifecyclePhase.TEST, lifecycle = "rps-rpp-vms")
+public class RpsRppVmsMojo extends RppVmsMojo {
+    public void execute() throws MojoExecutionException {
+        super.monitorFile = MonitorMojo.MONITOR_FILE;
```

Comment:
make sure this is renamed to reflect the fact that it is an agent config file, not a monitor file